src/main/java/com/netuitive/ananke/statsd/client/request/SetRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.netuitive.ananke.statsd.client.request;
+
+import com.netuitive.ananke.statsd.entity.Tag;
+import java.util.List;
+import lombok.Data;
+import lombok.experimental.Wither;
+
+/**
+ *
+ * @author john.king
+ */
+@Data
+@Wither
+public class SetRequest {
```

Comment:
it appears a lot of Requests shared common things like metric, sampleRate, we can define an abstract class, i.e. AbstractStatsDRequest.